website/src/components/examples-sandbox.tsx

```diff
@@ -0,0 +1,158 @@
+  hidenavigation: '1',
+  view: 'editor',
+  module: '%2F.meshrc.yml',
+};
+
+export interface ExamplesSandboxProps extends React.HTMLAttributes<HTMLElement> {
+  preventStealingFocusWithUnpleasantDelay?: boolean;
+  lazy?: boolean;
+}
+
+export function ExamplesSandbox({
+  preventStealingFocusWithUnpleasantDelay,
+  lazy = false,
+  ...rest
+}: ExamplesSandboxProps) {
+  const [exampleDir, setExampleDir] = useState('json-schema-example');
+  const [isVisible, setIsVisible] = useState(lazy ? false : true);
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const observer = new IntersectionObserver(
+      entries => {
+        entries.forEach(entry => {
+          if (entry.isIntersecting) {
+            setIsVisible(true);
+            observer.disconnect();
+          }
+        });
+      },
+      {
+        rootMargin: '100px',
+        threshold: 0.1,
+      },
+    );
+
+    if (containerRef.current) {
+      observer.observe(containerRef.current);
+    }
+
+    return () => observer.disconnect();
+  }, []);
+
+  const repo = 'ardatan/graphql-mesh';
+
+  const src = `https://codesandbox.io/embed/github/${repo}/tree/master/examples/${exampleDir}?${new URLSearchParams(CODESANDBOX_OPTIONS).toString()}`;
+
+  return (
+    <div {...rest} className={cn('w-full', rest.className)}>
+      <div className="flex items-center justify-center gap-2">
+        Choose Live Example:
+        <select
+          value={exampleDir}
+          onChange={e => {
+            setExampleDir(e.target.value);
+          }}
+          className="bg-inherit hive-focus w-[200px] cursor-pointer px-3 pr-8 p-2 border-beige-400 dark:border-neutral-800 border rounded-lg hover:bg-beige-100 dark:hover:bg-neutral-900 appearance-none"
+        >
+          {Object.entries(EXAMPLES).map(([groupName, group]) => (
+            <optgroup key={groupName} label={groupName}>
+              {Object.entries(group).map(([exampleName, value]) => (
+                <option key={exampleName} label={exampleName} value={value}>
+                  {exampleName}
+                </option>
+              ))}
+            </optgroup>
+          ))}
+        </select>
+        <div className="w-0">
+          <CaretSlimIcon className="size-4 relative pointer-events-none -left-9" />
+        </div>
+      </div>
+      <div
+        ref={containerRef}
+        className="w-full mt-8 h-[520px] sm:h-[720px] bg-beige-100 dark:bg-[#0D0D0D]"
+      >
+        {isVisible && (
+          <iframe
+            loading={lazy ? 'eager' : 'lazy'}
+            src={src}
+            className="w-full h-full"
+            title={exampleDir}
+            allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media; usb"
+            sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"
+            style={{ display: preventStealingFocusWithUnpleasantDelay ? 'none' : 'block' }}
```

Comment:
```suggestion
        cn(preventStealingFocusWithUnpleasantDelay && 'hidden')
```
you can use just class?